### ISSUE TITLE
 Uncaught Error: Class "phpseclib3\Net\SFTP" not found

### ISSUE THREAD

**Issue by MrGoatsy** [phpseclib/phpseclib#1598]
I downloaded the latest release and after following the first step on this page:
https://phpseclib.com/docs/sftp

I got this error:
Uncaught Error: Class "phpseclib3\Net\SFTP" not found

**Comment by terrafrost** [comment]
You should use [Composer](https://getcomposer.org/) to install phpseclib v3

**Comment by beanno1116** [comment]
I used composer to install it and get the same error message

**Comment by terrafrost** [comment]
Do you have `require __DIR__ . '/vendor/autoload.php';` at the top of your PHP file?

**Comment by kuldeep-ssense** [comment]
Yes, I did that but still got the same error. Attaching the snapshot for reference. 

![Screen Shot 2023-02-21 at 11 42 06 AM](https://user-images.githubusercontent.com/120278088/220406813-8772ada1-46a6-4874-ad73-f50f8ae934ec.png)

![Screen Shot 2023-02-21 at 11 42 28 AM](https://user-images.githubusercontent.com/120278088/220406834-56bcf233-6805-4f84-b4ed-8da36e36edcc.png)

**Comment by terrafrost** [comment]
phpseclib is namespaced. You need to `new \phpseclib3\Net\SFTP` and `new \phpseclib3\Crypt\RSA`. Or you could do `use phpseclib3\Net\SFTP` at the top.

**Comment by SonnyRod** [comment]
Same happening for me, I already remove the phpseclib/phpseclib through composer then require it again.

![Image](https://github.com/user-attachments/assets/c1fdc65f-0328-431c-982f-5b718deb0e1f)
